### PR TITLE
Patch release 4.65.1

### PR DESCRIPTION
**Pull request by dnil**
This PR marks a new Scout release. We apply semantic versioning. This is a patch release since it contains smaller fixes to existing features and bug corrections.

### Fixed
- [x] Visibility of `Gene(s)` badges on SV VariantS page
- [x] Hide dismiss bar on SV page not working well
- [x]  Delivery report PDF download
- [x]  Saving Pipeline version file when loading a case

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by DN, CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Tested `Saving Pipeline version file when loading a cas`e with this case: https://scout-stage.scilifelab.se/cust000/15026-mip7

I didn't have a proper exe file so I've loaded another yaml

**Comment by dnil** [comment]
Hi, looks good with the yaml! Which variant of load was this, just a report load or a full update or reload from scratch after delete - or even all of above?

**Comment by dnil** [comment]
The more recent verification reruns would have both yamls, say `safeguinea`, if you want any more for load/reload! The `*_vcol.yaml` is the one noted as `exe_ver` in housekeeper, and the `reference_info.yaml` should be obvious. Then I guess they will not yet be in the `scout_load.yaml` from `cg`, or did I miss a PR there? 😊

**Comment by northwestwitch** [comment]
> The more recent verification reruns would have both yamls, say `safeguinea`, if you want any more for load/reload! The `*_vcol.yaml` is the one noted as `exe_ver` in housekeeper, and the `reference_info.yaml` should be obvious. Then I guess they will not yet be in the `scout_load.yaml` from `cg`, or did I miss a PR there?

Ah Ok! No no more likely me to have missed a PR. But I think for testing purposes any yaml would no, no?

**Comment by dnil** [comment]
> > The more recent verification reruns would have both yamls, say `safeguinea`, if you want any more for load/reload! The `*_vcol.yaml` is the one noted as `exe_ver` in housekeeper, and the `reference_info.yaml` should be obvious. Then I guess they will not yet be in the `scout_load.yaml` from `cg`, or did I miss a PR there?
> 
> 
> 
> Ah Ok! No no more likely me to have missed a PR. But I think for testing purposes any yaml would no, no?

Yes, nothing magical in there - any yaml via load is fine!

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
-## [unreleased]
+## [4.65.1]
```
Comment (northwestwitch) [inline CHANGELOG.md:7]: 👍🏻

**Review by northwestwitch — APPROVED** [review]
All good!

**Comment by northwestwitch** [comment]
Deployed on Hasta prod:

image

**Comment by dnil** [comment]
Deployed to `cg-prod-services` as well!
```
[hiseq.clinical@cg-prod-services ~]$ systemctl --user stop scout@v4.65; systemctl --user start scout@v.4.65.1
```
Screenshot 2023-03-29 at 13 42 41

**Pull request merged**